packages/talos_agent/src/messaging/kafka.rs

```diff
@@ -54,7 +54,9 @@ impl KafkaPublisher {
         cfg.set("bootstrap.servers", &kafka.brokers)
             .set("message.timeout.ms", &kafka.message_timeout_ms.to_string())
             .set("queue.buffering.max.messages", "1000000")
-            .set("topic.metadata.refresh.interval.ms", "4")
+            .set("topic.metadata.refresh.interval.ms", "5000")
+            .set("socket.keepalive.enable", "true")
+            .set("acks", "0")
```

Comment:
We probably shouldn't put `acks=0` for real runs?

Reply:
I don't believe this is a problem.
The implication of `acks=0` is that a message may appear to be published to the client (i.e., agent) whereas it hasn't been stably recorded on the broker(s), possibly losing the message as a result.
The effect is that the decision will never come, for lack of candidate, and the agent will retry as of the eventual timeout.

Reply:
Has this branch passed the review?